docker-compose.yml

```diff
@@ -23,15 +23,17 @@ services:
       - RASA_URL=${RASA_URL:-http://rasa:5000}
       - DUCKLING_URL=${DUCKLING_URL:-http://duckling:8000}
       - MODEL_DIR=${MODEL_DIR:-./local-storage/rasa/nlu-model}
+      - DOCKER_HOST=${DOCKER_HOST:-localhost}
+      - SWAGGER_HOST=${SWAGGER_HOST:-localhost}:8000
```

Comment:
@dcalvom this one has a default of localhost:8000 in the code right? So there's no need to set it here?

Reply:
@btotharye why is the port outside of the parameter?

Reply:
to make it easier on the end user so in the .env file they just have to put a IP address.